docs/getting-started/using-the-substrate-scripts.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Using the Substrate node and module setup scripts"
+title: "Using the Substrate Scripts"
```

Comment:
I like your new title better than the old one. I would like `Using the Substrate-Up Scripts` even better because

1. That's what the repo is called
2. It disambiguates substrate-up scripts from scripts/init.sh and scripts/build.sh

Up to you whether it's worth revising.